src/game-status/adiMalka.js

```diff
@@ -0,0 +1,28 @@
+const optionToGuess = 4;
+
+/**
+ * thr function get array of strung - colors and return random array of new options
```

Comment:
the function gets an array of colors (strings) and generates an array that the user needs to guess, right? 
if that's the case, just retype the description : the function gets an array of strings (colors) and returns random array of colors for the user to guess